sig-scalability/slos/slos.md

```diff
@@ -127,6 +127,10 @@ __TODO: Cluster churn should be moved to scalability thresholds.__
 sliding window. However, for the purpose of SLO itself, it basically means
 "fraction of good minutes per day" being within threshold.
 
+<a name="footnote2">\[2\]</a> This SLI only measures latency incurred by the processing
```

Comment:
I would remove it from here - there are a bunch of other footnotes in details so let's not propagate only a subset of those to here. If someone wants more detail, they should visit the other page.